dsref/version_info.go

```diff
@@ -64,13 +67,31 @@ type VersionInfo struct {
 	//
 	// Timestamp field from the commit
 	CommitTime time.Time `json:"commitTime,omitempty"`
+	// Title field from the commit
+	CommitTitle string `json:"commitTitle,omitempty"`
+	// Message field from the commit
+	CommitMessage string `json:"commitMessage,omitempty"`
 	//
 	// About the dataset's history and location
 	//
 	// Number of versions that the dataset has
 	NumVersions int `json:"numVersions,omitempty"`
 	// FSIPath is this dataset's link to the local filesystem if one exists
 	FSIPath string `json:"fsiPath,omitempty"`
+	//
+	// Run Fields come from applying a script during version creation
+	//
+	// RunID from either Commit.RunID, or the ID of a failed run when no
+	// Path value (version is present)
+	RunID string `json:"runID,omitempty"`
+	// RunStatus is a string version of the run.Status enumeration. This value will
+	// always be one of:
+	//    ""|"waiting"|"running"|"succeeded"|"failed"|"unchanged"|"skipped"
+	// it's a type a string to follow the "plain old data" pattern
+	RunStatus string `json:"runStatus,omitempty"`
```

Comment:
The conversion functions copy RunID from the Commit component. Where do RunStatus and RunDuration come from? Are they not on the commit? Either way should noted on the definition here.